extensions/guacamole-auth-quickconnect/src/main/java/org/apache/guacamole/auth/quickconnect/conf/ConfigurationService.java

```diff
@@ -21,7 +21,6 @@
 
 import com.google.inject.Inject;
 import java.util.Collection;
-import java.util.List;
```

Comment:
We should probably avoid doing whitespace/import fixes for files not otherwise modified for this change, to keep the git history cleaner.

Reply:
Sounds good. I'll just collect all the little fix-ups I'm accumulating and open a Jira issue for that.